ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -1210,6 +1219,26 @@ extension BottomCommandingController: CommandingItemDelegate {
             break
         }
     }
+
+    func commandingItem(_ item: CommandingItem, didChangeHiddenTo value: Bool) {
+        guard let view = itemToBindingMap[item]?.view else {
+            return
+        }
+
+        switch view {
+        case let tabBarItemView as TabBarItemView:
+            if tabBarItemView.isHidden != value {
+                tabBarItemView.isHidden = value
+            }
+        case let cell as TableViewCell:
+            if cell.isHidden != value {
+                cell.isHidden = value
+                tableView.reloadData()
+            }
+        default:
```

Comment:
Is this ok to hit the default case? Should be an error of some sort?

Reply:
This code isn't needed with the most recent changes.